easybuild/easyconfigs/p/Python/Python-3.9.6-GCCcore-11.2.0-bare.eb

```diff
@@ -0,0 +1,34 @@
+name = 'Python'
+version = '3.9.6'
+versionsuffix = '-bare'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['d0a35182e19e416fc8eae25a3dcd4d02d4997333e4ad1f2eee6010aadc3fe866']
+
+builddependencies = [
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('binutils', '2.37'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.36'),
+    ('XZ', '5.2.5'),
+    ('libffi', '3.4.2'),
+    ('OpenSSL', '1.1', '', True),
```

Comment:
I suspect we don't need all of these for the bare version? Or do we?

Reply:
Ok looking for a build log for 3.9.5 it does seem to do have all that stuff int he base python. I didn't expect sqlite